cmd/manager/main.go

```diff
@@ -161,7 +158,12 @@ func main() {
 	}
 
 	cl := mgr.GetClient()
-	catalogClient := catalogclient.New(cl, cache.NewFilesystemCache(cachePath, httpClient))
+	catalogsCachePath := filepath.Join(cachePath, "catalogs")
+	if err := os.MkdirAll(catalogsCachePath, 0700); err != nil {
```

Comment:
Nothing that the previous mode (for `bundles`) was `0755`, does this make a difference?

Reply:
It shouldn't. We don't expect any other users or groups to read these files.